client/app/components/Toastr/index.js

```diff
@@ -0,0 +1,69 @@
+import React, { Component } from 'react';
+import { ToastContainer, ToastMessage } from 'react-toastr';
```

Comment:
Let's try to stick to the inspinia theme. React-toastr may be deviating from Inspinia toastr. use toastr.js provided by inspinia